designsafe/templates/menu/breadcrumb.html

```diff
@@ -1,5 +1,10 @@
 {# https://github.com/django-cms/django-cms/blob/release/3.11.x/menus/templates/menu/breadcrumb.html #}
+{% load cms_tags %}
+
 {% for ance in ancestors %}
+{% page_attribute "slug" ance.id as current_slug %}
+{% page_attribute "slug" ance.parent_id as parent_slug %}
+{% if not parent_slug|truncatechars:17 == "tools-applicatio…" or current_slug == "tools-applications-new" %}
```

Comment:
**Translation:**

1. If parent page starts with "Tools & Applications" (e.g. an app category), then skip rendering breadcrumb.
2. **But** if current page is "Tools & Applications (New)" placeholder page hierarchy, then do render breadcrumb.

#2 prevents skipping "Tools & Applications (New)" whose parent page starts with "Tools & Applications".

Reply:
I would have used `reverse_id`[^1], but I found no way via template to access parent `reverse_id` 😠 and I thought a `templatetag`[^2]

[^1]: how to identify page; does not change if page slug changes ([1](https://docs.django-cms.org/en/latest/reference/templatetags.html#page-lookup), [2](https://docs.django-cms.org/en/latest/reference/api_references.html#module-cms.api))
[^2]: how to write custom template tags or filters with Python